### DISCUSSION TITLE
about host name of HTTPScaledObject

### DISCUSSION THREAD

**Discussion by georgetor2020** [kedacore/http-add-on#1122] [General]
I install keda and keda-http-add-on on Openshift 4.13. How to setup host name for HTTPScaledObject & Ingress so I can access from url? 

The hello-openshift.apps.ocp.test.ca sound does not work and get error:
Application is not available
---
apiVersion: v1
kind: Service
metadata:
  name: hello
  labels:
    app: hello
spec:
  type: ClusterIP
  ports:
    - port: 80
      targetPort: 8080
      protocol: TCP
      name: http
  selector:
    app: hello
---
kind: Service
apiVersion: v1
metadata:
  name: keda-add-ons-http-interceptor-proxy
spec:
  type: ExternalName
  externalName: keda-add-ons-http-interceptor-proxy.keda.svc.cluster.local
---                 
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: hello-ingress
spec:
  ingressClassName: nginx
  rules:
  - host: hello-openshift.apps.ocp.test.ca
    http:
      paths:
      - path: /
        pathType: Prefix
        backend:
          service:
            name: keda-add-ons-http-interceptor-proxy
            port:
              number: 8080

---              
kind: HTTPScaledObject
apiVersion: http.keda.sh/v1alpha1
metadata:
    name: hello-scaledobject
    path: /hello
    namespace: keda-test
spec:
    hosts:
        - hello-openshift.apps.ocp.test.ca
    scaleTargetRef:
        name: hello
        kind: Deployment
        apiVersion: apps/v1
        service: hello
        port: 80
    replicas:
        min: 1
        max: 4
    scalingMetric:
        requestRate:
            granularity: 1s
            targetValue: 5
            window: 1m

**Comment by linkvt** [comment]
Closing as outdated — this predates several releases and either got resolved since or never reached a conclusion. See current docs (https://keda.sh/http-add-on/0.15/) or feel free to open a new discussion if still relevant.
